kie-wb/kie-wb-distribution-wars/src/main/assembly/component-kie-wb-tomcat-7-common.xml

```diff
@@ -96,6 +96,7 @@
 
             <!-- Errai -->
             <exclude>WEB-INF/lib/errai-jboss-as-support-*.jar</exclude>
+            <exclude>WEB-INF/lib/errai-cdi-server-*.jar</exclude>
```

Comment:
Thanks, that is the piece I was missing.

See https://github.com/droolsjbpm/kie-wb-distributions/pull/267 which is about upgrading Weld from 1.x to 2.x. So when that is merged this exclusion should not be needed anymore.